reusable_learning_app/lib/screens/Content/settings_body.dart

```diff
@@ -13,18 +14,261 @@ class SettingsBody extends StatefulWidget implements NavItem   {
+                                          fontSize: 16, color: Colors.black)),
+                                  Padding(
+                                    padding: const EdgeInsets.only(right: 13),
+                                    child: DropdownButton<String>(
+                                      value: _dropdownValue,
+                                      elevation: 16,
+                                      style: const TextStyle(
+                                          color: Colors.deepPurple),
+                                      underline: Container(
+                                        height: 2,
+                                        color: Colors.deepPurpleAccent,
+                                      ),
+                                      onChanged: (String? newValue) {
+                                        setState(() {
+                                          _dropdownValue = newValue!;
+                                        });
+                                      },
+                                      items: <String>['English', 'Russian']
+                                          .map<DropdownMenuItem<String>>(
+                                              (String value) {
+                                        return DropdownMenuItem<String>(
+                                          value: value,
+                                          child: Text(value),
+                                        );
+                                      }).toList(),
+                                    ),
+                                  )
+                                ],
+                              ))
+                        ],
+                      )))) // General
+          ,
+          Padding(
+              padding: const EdgeInsets.only(left: 7, right: 7, bottom: 11),
+              child: Card(
+                  shape: RoundedRectangleBorder(
+                    borderRadius: BorderRadius.circular(15.0),
+                  ),
+                  color: Colors.white,
+                  elevation: 10,
+                  child: Padding(
+                    padding: const EdgeInsets.only(
+                        left: 17, top: 5, bottom: 5, right: 5),
+                    child: Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: [
+                        const Text("Audio",
+                            style:
+                                TextStyle(fontSize: 16, color: Colors.black)),
+                        Switch(
+                            value: _isAudioOn,
+                            activeColor: Colors.deepPurple,
+                            onChanged: (value) {
+                              _changeAudioMode(value);
+                            })
+                      ],
+                    ),
+                  ))) // Audio
+          ,
+          Padding(
```

Comment:
Make "notification" and "get support" options clickable not only with arrow icon.

<img width="310" alt="image" src="https://user-images.githubusercontent.com/62881333/174034058-1d05d445-56cb-4a52-81bd-4a62170cc443.png">

Reply:
Done